common/src/main/java/apoc/ApocConfig.java

```diff
@@ -140,13 +154,34 @@ public Configuration getConfig() {
         return config;
     }
 
+    private String evaluateIfCommand(String settingName, String entry) {
+        if (Config.isCommand(entry)) {
+            Preconditions.checkArgument(
+                    expandCommands,
+                    format(
+                            "%s is a command, but config is not explicitly told to expand it. (Missing --expand-commands argument?)",
+                            entry));
+            String str = entry.trim();
+            String command = str.substring(2, str.length() - 1);
```

Comment:
Does neo4j only admit command expansion with `$(command)` or is there other way? I'm asking because in bash I think you can also enclose in backticks:

These two produce the same result:

```
echo hola $(echo 'adios')
```

```
echo hola `echo 'adios'`
```